GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -188,14 +177,6 @@ public RevisionGrid()
             IsMessageMultilineDataGridViewColumn.Resizable = DataGridViewTriState.False;
 
             this.HotkeysEnabled = true;
-            try
```

Comment:
Just to be clear: Where is this run now?

Reply:
That's the thing - this call is not needed.

Currently the call to `SetRevisionsLayout(RevisionGridLayout)` is required to initialise the current layout (stored in `_layout` variable).
In the proposal we no longer deal with layouts but use renderers instead, and the current renderer is loaded on line 121: `_layoutRenderer = _layoutRendererFactory.GetCurrent();`. The the rendering routines (e.g. `OnCellDraw` etc) use the current renderer.

Reply:
One can say that the current renderer is again retrieved in `SetRevisionsLayout` but the current implementation requires the knowledge of the current layout in the constructor (can't tell you from the top of my head). Hence the constructor makes this call.
This can probably be untangled in further refactors....